lib/adagrams.rb

```diff
@@ -0,0 +1,136 @@
+require 'pry'
+
+#create data structure
+all_letters = []
+
+letters = {"A" => 9, "B" => 2, "C" => 2, "D"=> 4, "E" => 12, "F"=> 2, "G"=>3, "H"=>2, "I" => 9, "J"=>1, "K"=> 1, "L"=>4, "M"=> 2, "N"=> 6, "O"=>8, "P" => 2, "Q" => 1, "R"=>6, "S"=>4, "T"=>6, "U"=>4, "V"=>2, "W"=>2, "X"=>1, "Y"=>2, "Z"=>1}
+
+
+# Fill pool of letters array with letters
+letters.each do |letter, freq|
+  count = 0
+  until count == freq
+    all_letters << letter
+    count += 1
+  end
+end
+
+
+# Define method for creating user hand
+def draw_letters(array)
+  user_letters = array.sample(10)
+  return user_letters
+end
+
+DRAWN_LETTERS = draw_letters(all_letters)
```

Comment:
I think there was some confusion about what was expected here. The idea is that `draw_letters` should be called by the driver code (`wave-X-game.rb`) or by the specs in order to create a hand of letters. Your code should not ever need to call `draw_letters`.